content/maxwell3_fdem/sphere_freespace/excitation_factor.rst

```diff
@@ -0,0 +1,135 @@
+.. _excitation_factor
+
+Excitation Factor for Special Cases
+-----------------------------------
+
+**Purpose**: The characteristic dipole response from a conductive and magnetically permeable sphere is defined by its excitation factor.
+Here, we present analytic expressions and discuss the excitation factors for several specific cases, including: a conductive and magnetically permeable sphere, a purely conductive sphere, and the zero-frequency response from a highly permeable sphere.
+
+
+Conductive and Magnetically Permeable Sphere
+============================================
+
+
+According to Eq. :eq:`eqExcitationFactor`, the excitation factor for a conductive and magnetically permeable sphere is given by:
+
+.. math::
+	\chi (i \omega) = \frac{3}{2} \Bigg [ \! \frac{2\mu \big [ tanh(\alpha) - \alpha  \big ] + \mu_0 \big [\alpha^2 \, tanh(\alpha) - \alpha + tanh(\alpha) \big ] }{\mu  \big [ tanh(\alpha) - \alpha \big ] - \mu_0 [ \alpha^2 \, tanh(\alpha) - \alpha + tanh(\alpha) \big ] } \! \Bigg ]
+	:label: ChiApprox
+
+where
+
+.. math::
+	\alpha = \Big [ i\omega\mu\sigma \Big ]^{1/2}R
+
+
+.. figure::
+	./images/figChiOmega.png
+	:figwidth: 45%
+	:align: right
+	:name: ChiOmega
+
+	Magnetization factor for a conductive and magnetically permeable sphere in free-space with :math:`R` = 25 m, :math:`\sigma` = 10 S/m, and :math:`\mu` = 1.1 :math:`\mu_0`.
```

Comment:
"Excitation" (not Magnetization?) and again, a few annotations, would be helpful